validation/src/test/scala/hmda/validation/rules/lar/validity/V285Spec.scala

```diff
@@ -0,0 +1,47 @@
+package hmda.validation.rules.lar.validity
+
+import hmda.validation.dsl.{ Failure, Success }
+import hmda.validation.rules.lar.LarEditCheckSpec
+import org.scalacheck.Gen
+
+class V285Spec extends LarEditCheckSpec {
+
+  property("Succeeds when state is valid FIPS code") {
+    forAll(larGen) { lar =>
+      whenever(lar.geography.state != "NA") {
+        V285(lar) mustBe a[Success]
+      }
+    }
+  }
+
+  property("Fails when state is invalid FIPS code") {
+    forAll(larGen, badStateGen) { (lar, state) =>
+      whenever(lar.geography.state != "NA") {
+        val invalidGeography = lar.geography.copy(state = state)
+        val invalidLar = lar.copy(geography = invalidGeography)
+        V285(invalidLar) mustBe a[Failure]
+      }
+    }
+  }
+
+  property("Succeeds when state is NA and MSA/MD is NA") {
+    forAll(larGen) { lar =>
+      val validGeography = lar.geography.copy(state = "NA", msa = "NA")
+      val validLar = lar.copy(geography = validGeography)
+      V285(validLar) mustBe a[Success]
+    }
+  }
+
+  property("Fails when state is NA but MSA/MD is not NA") {
+    forAll(larGen) { lar =>
+      whenever(lar.geography.msa != "NA") {
```

Comment:
@schbetsy This should not be an issue. The CBSA lookup table uses the same list of states, while the [MSA](http://www.census.gov/population/metro/) is defined by [OMB](https://www.whitehouse.gov/omb/inforeg_statpolicy) the list of geographies that conform them is a list of counties, identified by their FIPS code, i.e. example [here](http://www.census.gov/population/metro/files/lists/2015/List1.xls)